expo/gbe/report_views.py

```diff
@@ -165,14 +181,30 @@ def env_stuff(request, conference_choice=None):
 def personal_schedule(request, profile_id='All'):
     viewer_profile = validate_perms(request, 'any', require=True)
 
+    conference_slugs = conf.Conference.all_slugs()
+    if request.GET and request.GET.get('conf_slug'):
+        conference = conf.Conference.by_slug(request.GET['conf_slug'])
+    else:
+        conference = conf.Conference.current_conf()
+
     if profile_id == 'All':
         people = conf.Profile.objects.all().select_related()
     else:
-        people = []
+        people = []  #  Set it to be self, in list format
+
+    tmp_peeps = []
+    for peep in range(0,len(people)):
+        for vent in range(0, len(people[peep].schedule)):
+            if conference.conference_slug == people[peep].schedule[vent].eventitem.get_conference().conference_slug:
+                tmp_peeps.append(people[peep])
+                break  #  Yes, I know this is bad form, refactor later
+    people = tmp_peeps
+    del tmp_peeps
 
     return render(request,
                   'gbe/report/printable_schedules.tmpl',
-                  {'people': people})
+                  {'people': people,
+                   'conference_slugs': conference_slugs,  'conference': conference})
```

Comment:
Please maintain dicts as either horizontal (leave them on one line) or vertical (separate each item on its own line). This is messy.